### PR TITLE
package.json indentation fixed 

### LINKED ISSUE #4056: Fix package.json closing bracket indentation
difficulty/newcomer 

Opened via [gitter](https://gitter.im/coala/coala/?at=58e775bb8fcce56b20dbe107) by @jayvdb

### PR DESCRIPTION
**Pull request by aashutoshrathi**
package.json : fixed second last bracket's indentation

Removed extra spaces from package.json

Fixes #4056 

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by arjunsinghy96** [comment]
@aashutoshrathi 
I think you are not a coala member yet. Just go to our [chat](https://gitter.im/coala/coala) and say hello. Someone will invite you as a newcomer. Accept the invitation and you are good to go.

File: package.json [hunk 1]
```diff
@@ -2,5 +2,5 @@
   "name": "coala",
   "dependencies": {
     "csslint": "~1"
-    }
+   }
```
Comment (jayvdb) [inline package.json]: this bracket is not correctly indented.  Remove another leading space.

**Comment by aashutoshrathi** [comment]
@jayvdb take a look now 👍

**Comment by arjunsinghy96** [comment]
Please squash the two commits and make them one.(use `git rebase -i HEAD~2`) .Also, your commit message is still not according to our guidelines. Please read the [docs](https://api.coala.io/en/latest/Developers/Writing_Good_Commits.html). An issue link is required. If there is no issue, you can create one.

**Comment by arjunsinghy96** [comment]
I would recommend reading our [newcomer's guide](http://api.coala.io/en/latest/Developers/Newcomers_Guide.html).

**Comment by aashutoshrathi** [comment]
@arjunsinghy96 please take a look now

**Comment by adhikasp** [comment]
Hi @aachiritoaei 
The code itself is already OK. But the commit message is still lacking 😅 
Make sure you follow this guideline https://api.coala.io/en/latest/Developers/Writing_Good_Commits.html

And also, you shouldn't change PR issue back and forth like that. You can use `cobot` command in our [gitter](https://gitter.im/coala/coala) channel to mark WIP.

**Comment by aashutoshrathi** [comment]
@adhikasp i hope you are referring to me instead of @aachiritoaei

**Comment by adhikasp** [comment]
Oh... yeah... edited 😖

**Comment by aashutoshrathi** [comment]
@jayvdb take a look now

**Comment by arjunsinghy96** [comment]
For rebasing:
1. `git checkout master`
2. `git fetch upstream`
3. `git rebase upstream/master`
4. `git checkout <branch_name>`
5. `git rebase master`

You need to reset last commit before this.

For amending commit message:
1. `git commit --amend` This will amend your last commit message.

Please, please read the guidelines [here](https://api.coala.io/en/latest/Developers/Writing_Good_Commits.html).

**Comment by jayvdb** [comment]
You have three commits on https://github.com/coala/coala/pull/4063/commits

We want one.

Read all the messages you have received above on this Pull Request.
If you cant understand them, come and [chat](http://coala.io/chat) for a real human to guide you.

Or, close this PR because we will NOT accept three commits for a one line change.
Newcomer issues are part of our process, so newcomers can learn to use git properly, otherwise they are wasting our time.

**Comment by aashutoshrathi** [comment]
@jayvdb they said above that this commit is not required that's why  I didn't merged the changes

**Comment by jayvdb** [comment]
You are not reading correctly.

Unassigned.  Please come back to coala when you're ready to learn from our guides.

**Pull request closed without merging**